src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/JsonTest/JsonTest.cs

```diff
@@ -67,8 +67,7 @@ public void TestJsonWrite()
         {       
             string tableName = DataTestUtility.GetUniqueNameForSqlServer("Json_Test");
```

Comment:
Is there a scenario for which we may need unique name for `Json_test` and `spJson_WriteTest`? If not, we can avoid it. In case of test failure, we may end up with a leftover table and stored proc with unique name.

Reply:
Done